src/lib/cost_model/cost_feature.cpp

```diff
@@ -0,0 +1,27 @@
+#include "cost_feature.hpp"
+
+#include "utils/assert.hpp"
+
+namespace opossum {
```

Comment:
These particular ones *could* be changed. I started using Assert over DebugAssert because of reasons explained here: https://github.com/hyrise/hyrise/pull/814#discussion_r184335348

Reply:
The question is whether these functions are performance critical. As they might be executed for every query plan multiple times, I would assume they are.

Reply:
yeah, but they are like 0 overhead, too and while the function is called multiple times it's not in a hot loop.